File: chart/src/main/java/cn/jingzhuan/lib/chart/event/OnScaleListener.java
package cn.jingzhuan.lib.chart.event;

import cn.jingzhuan.lib.chart.Viewport;

/**
 * @since 2023-02-26
 */
public interface OnScaleListener {

    void onScaleStart(Viewport viewport);

    void onScale(Viewport viewport);

    void onScaleEnd(Viewport viewport);

}

File: chart/src/main/java/cn/jingzhuan/lib/chart3/event/OnScaleListener.kt
package cn.jingzhuan.lib.chart3.event

import cn.jingzhuan.lib.chart3.Viewport


/**
 * @since 2023-09-10
 * created by lei
 * 缩放监听
 */
interface OnScaleListener {
    fun onScaleStart(viewport: Viewport)
    fun onScale(viewport: Viewport)
    fun onScaleEnd(viewport: Viewport)
}